config/sidekiq.yml

```diff
@@ -3,18 +3,17 @@
 :concurrency: 32
 :logfile: ./log/sidekiq.json.log
 :timeout: 4
-
 # Any changes to the queues should be reflected in `docs/queues.md`.
+# https://github.com/mperham/sidekiq/wiki/Advanced-Options#queues
 :queues:
   - delivery_transactional
-  - [delivery_immediate_high, 10]
-  - [delivery_immediate, 5]
-  - [process_and_generate_emails, 4]
-  - [default, 3]
-  - [delivery_digest, 2]
-  - [email_generation_digest, 1]
+  - delivery_immediate_high
+  - delivery_digest
+  - delivery_immediate
+  - process_and_generate_emails
```

Comment:
I think this means that if there are any non high priority emails in the queue to be sent, no content changes will be processed even if that content change is a high priority one (for travel advice for example). The weighting means we can still generate emails for high priority content as it comes in, and they can sort of skip the queue and go straight to the high priority queue (which delays non high priority content, but that's okay).